ledger/acctupdates_test.go

```diff
@@ -845,3 +845,161 @@ func TestAcctUpdatesDeleteStoredCatchpoints(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 0, len(fileNames))
 }
+
+// listAndCompareComb lists the assets/applications and then compares against the expected
+// It repeats with different combinations of the limit parameters
+func listAndCompareComb(t *testing.T, au *accountUpdates, expected map[basics.CreatableIndex]modifiedCreatable) {
+
+	// test configuration parameters
+
+	// pick the second largest index for the app and asset
+	// This is to make sure exactly one element is left out
+	// as a result of max index
+	maxAss1 := basics.CreatableIndex(0)
+	maxAss2 := basics.CreatableIndex(0)
```

Comment:
I don't see where you're using `maxAss2` and `maxApp2`. 
Also, I would have implemented this as a `map[CreatableType]basics.CreatableIndex`, so that if we extend the types of creatables, it would work as well.

Reply:
maxAss2 is used to keep track of the last two largest elements. 
It will be used more in my next update. 

Will postpone the implementation of the map for now. It makes sense in the test, but addition of new types will impact many other interfaces.